nuxeo-web-ui-ftest/test/pages/ui/selection.js

```diff
@@ -8,4 +8,12 @@ export default class Selection extends BasePage {
     this.el.element('nuxeo-clipboard-documents-button').click();
     this.waitForNotVisible();
   }
+
+  moveDown() {
+    this.el.element('nuxeo-move-documents-down-button').click();
```

Comment:
Would put a `this.el.waitForVisible('nuxeo-move-documents-down-button')` before this line to ensure we wait for the button to be there.